### PR TITLE
Add/lang database

### PR DESCRIPTION
**Pull request by rkimera94**

### CODE REVIEW

**Comment by rkimera94** [comment]
Still failing some tests, let me check it

File: database/things/fortran.pldb [hunk 1]
```diff
@@ -211,6 +211,7 @@ indeedJobs fortran engineer
  2017 447
 linkedInSkill fortran
  2018 148352
+annualReports https://fortran-lang.org/en/news/ officialBlogUrl
```
Comment (celtic-coder) [inline database/things/fortran.pldb:214]: Hi Kimera (@rkimera94),

Why is there "officialBlogUrl" at the end of this line? Should it be on a separate line?

Kind Regards,
Liam

File: database/things/java.pldb [hunk 2]
```diff
@@ -457,6 +457,9 @@ stackOverflowSurvey
   medianSalary 51888
   fans 17222
   percentageUsing 0.35
+annualReports https://www.oracle.com/java/technologies/javase/jdk-relnotes-index.html releaseNotesUrl
+annualReports https://blogs.oracle.com/java/releaseNotesUrl officialBlogUrl
+annualReports https://dev.java/community/events/ eventsPageUrl
```
Comment (celtic-coder) [inline database/things/java.pldb:462]: Again, these lines have "releaseNotesUrl", "officialBlogUrl" and "eventsPageUrl" at  the end of their lines. Is this a problem with wrapping at the end of a line?

**Comment by rkimera94** [comment]
hi Liam, @celtic-coder  I put those separate words on the line so that you can add different URLs put to indicate the type at the end of the line using the following: 

`"releaseNotesUrl", "officialBlogUrl" and "eventsPageUrl" `
Then when build based on the type: 
Screenshot 2022-10-27 at 17 09 29

**Comment by rkimera94** [comment]
So that we use on field annualReports- but at the end of the line we specify which URL type other than creating a field for each URL type @celtic-coder

**Comment by rkimera94** [comment]
so should I create different fields for each type?  release, Newsblog, report, events page @celtic-coder

**Comment by rkimera94** [comment]
@breck7 @celtic-coder  any suggestions? should I putt separate keywords?

**Comment by breck7** [comment]
One line per field please. So each of these things should be its own keyword.

**Pull request closed without merging**